package.json

```diff
@@ -28,9 +28,10 @@
     "prepublish": "yarn build"
   },
   "dependencies": {
-    "@trust/oidc-rp": "^0.3.0",
+    "@trust/oidc-rp": "^0.4.3",
     "auth-header": "^0.3.1",
-    "isomorphic-fetch": "^2.2.1"
+    "isomorphic-fetch": "^2.2.1",
+    "yarn": "^0.27.5"
```

Comment:
Please remove this line. Yarn is assumed to be the package manager, and it's not recommended to install it via `npm`.